src/test/java/fr/fifiz/training/app/java/own/OwmClientTest.java

```diff
@@ -0,0 +1,169 @@
+package fr.fifiz.training.app.java.own;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import fr.fifiz.training.app.java.owm.OwmClient;
+import fr.fifiz.training.app.java.owm.TechnicalException;
+import fr.fifiz.training.app.java.owm.WeatherResult;
+
+/**
+ * Unit test for OwmClient.
+ *
+ * @author bertrand
+ */
+public class OwmClientTest {
+  // Chemin de la ressource l'api current weather de OWN
+  private static final String WEATHER_API_PATH = "/current";
+  //
+  private static final String LOCAL_URL = "http://localhost:{port}{path}";
+  //
+  private static final String LOCAL_URL_MALFORMED = "http://localhost-{port}{path}";
+  //
+  private static final String LOCAL_URL_PATH = "{path}";
+  //
+  private static final String LOCAL_URL_PORT = "{port}";
+  //
+  private static final String OWN_RESULT_OK = "owm_ok.json";
+  //
+  private static final String OWN_RESULT_CITY_NOT_FOUND = "owm_citynotfound.json";
+
+  WireMockServer wireMockServer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/bdelion/basic-weather-app/pullRequest?prid=4408921)